apps/examples/wifi_manager/wm_test/wm_test_usage.h

```diff
@@ -34,11 +34,14 @@
 	" stress test 2: Unit TC and Integrated TC\n"											\
 	" stress test 3: scenario base TC (client)\n"						\
 	" stress test 4: scenario base TC (server)\n"						\
+	" stress test 5: DA Device simulation in Easy Setup process"                              \
 	"	 wm_test stress 1 [ssid] [security mode] [password]\n"			\
 	"	 wm_test stress 2 [ssid] [security mode] [password] [softap ssid] [softap password] [softap channel]\n" \
 	"	 wm_test stress 2 file [file path] [softap ssid] [softap security mode] [softap password] [softap channel]\n" \
 	"	 wm_test stress 3 [ssid] [security mode] [password] [softap ssid] [softap password] [softap channel]\n" \
 	"	 wm_test stress 4 [ssid] [security mode] [password] [softap ssid] [softap password] [softap channel]\n" \
+	"        wm_test stress 5 [ssid] [security mode] [password] [softap ssid] [softap password] [softap channel]\n"  \
```

Comment:
How about to change command. There're too many stress test. So I think following commands are easy to understand
TASH> wm_test easy_client [ssid]...
TASH> wm_test easy_server ...

Reply:
Changes Done as suggested.